core/src/main/java/ai/timefold/solver/core/impl/move/streams/dataset/common/bridge/AftBridgeUniDataStream.java

```diff
@@ -0,0 +1,46 @@
+package ai.timefold.solver.core.impl.move.streams.dataset.common.bridge;
+
+import java.util.Objects;
+
+import ai.timefold.solver.core.impl.bavet.common.TupleSource;
+import ai.timefold.solver.core.impl.move.streams.dataset.AbstractDataStream;
+import ai.timefold.solver.core.impl.move.streams.dataset.AbstractUniDataStream;
+import ai.timefold.solver.core.impl.move.streams.dataset.DefaultDataStreamFactory;
+import ai.timefold.solver.core.impl.move.streams.dataset.common.DataNodeBuildHelper;
+
+public final class AftBridgeUniDataStream<Solution_, A>
```

Comment:
What is the meaning of Aft?

Reply:
"Aft" and "fore" are english words, opposites of each other. "Fore" means in front of something, and "aft" means behind something.